### PR TITLE
LPS-123917 [Content Performance] Top referring sites & domains (referral traffic detail) 

### PR DESCRIPTION
**Pull request by dgarciasarai**
**Description**
We are going to add a second level (detail) for Referral traffic channel in Content Performance panel.

**Solution**
I added a detailed view of the Referral traffic channel. This will be hidden because this traffic channel does not conform to the checks to be displayed as a link (there is no `details` attribute within the traffic channel).

In this pull request, I only added the test `displays the referral detail according to API` but also organize the test file `Detail` with two describe sections to separate Organic and Referral tests.

* `referringDomains` will be an array of 5 URLs
* `referringPages` will be an array of 10 URLs

```js
{
	details: {
	  referringDomains: [{url: "http://www.example.com", traffic: 1234}],
	  referringPages: [{url: "http://www.example.com", traffic: 1234}],
	},
	helpMessage: 'This is the number of page views generated by people coming to your page from other sites which are not search engine pages or social sites.',
	name: 'referral',
	share: 10,
	title: 'Referral',
	value: 20,
}
```

**Screenshots**

Screenshot 2020-11-30 at 18 44 54 Screenshot 2020-11-30 at 18 45 17

### CODE REVIEW

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
ci:stop

**Comment by dgarciasarai** [comment]
ci:test:sf

**Comment by dgarciasarai** [comment]
ci:test:relevant

**Comment by dgarciasarai** [comment]
Hey @liferay-frontend, can you review this pull request and give me feedback? Thanks! 😃

File: modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/detail/ReferralDetail.js [hunk 1]
```diff
@@ -9,8 +9,216 @@
...
+							<ClayList.ItemField expand>
+								<ClayList.ItemText>
+									<ClayTooltipProvider>
+										<span
+											className="text-truncate-inline"
+											data-tooltip-align="top"
+											title={url}
+										>
+											<a
+												className="text-primary text-truncate"
```
Comment (dgarciasarai) [inline modules/dxp/apps/analytics/analytics-reports-web/src/main/resources/META-INF/resources/js/components/detail/ReferralDetail.js:190]: I should add a margin here too. I'll update the changes tomorrow and let the CI finish.

**Comment by dgarciasarai** [comment]
Any feedback here? 🤔  Thanks!

**Comment by cgoncas** [comment]
Hey @liferay-frontend!!!

The @liferay-tango team needs to have this pull merged in order to finish a critical issue.

We would love to hear your feedback, and we will apply it as soon as we receive it, but, for now, we are going to forward this pull.

Thanks!!

**Comment by cgoncas** [comment]
ci:forward

**Pull request closed without merging**